### ISSUE TITLE
Community Development Maintenance : 23cf956f-d416-4075-9deb-90ce3d3bdfb0

### ISSUE THREAD

**Issue by kyletaylored** [OpenDenton/City-of-Denton-Datasets#54]
**Title:** Community Development Maintenance
**ID:** 23cf956f-d416-4075-9deb-90ce3d3bdfb0
**Description:** Home Improvement Program - provides help to homeowners with minor urgent repairs. Stores structural damage to residential and commercial structures; and damage to public facilities collected during emergency response activities.
**Created:** 10 / 05 / 2016
**Updated:** 10 / 05 / 2016
**State:** active
**Organization:** City of Denton
**URL:** [http://data.cityofdenton.com/dataset/community-development-maintenance](http://data.cityofdenton.com/dataset/community-development-maintenance)
**Tags:**  Address,  Census Tract,  City,  Damage Assessment,  Date Minor Repair Closed,  Emergency,  Emergency Operations,  Income,  Income Amount,  Mobile,  Percent,  State,  Structual Damage,  Zip, Minor Repair Completed
**Groups:** Map Data

### Resources

- [ArcGIS Open Dataset](http://data.dentontxgis.opendata.arcgis.com/datasets/fcdc95d116c44587bb835bf3bbdfd56f_0), HTML
- [Esri Rest API](https://services.arcgis.com/9dTWrhPzuDPnUVXr/arcgis/rest/services/Community_Development_Maintenance/FeatureServer/0), Esri REST
- [GeoJSON](http://data.dentontxgis.opendata.arcgis.com/datasets/fcdc95d116c44587bb835bf3bbdfd56f_0.geojson), GeoJSON
- [CSV](http://data.dentontxgis.opendata.arcgis.com/datasets/fcdc95d116c44587bb835bf3bbdfd56f_0.csv), CSV
- [KML](http://data.dentontxgis.opendata.arcgis.com/datasets/fcdc95d116c44587bb835bf3bbdfd56f_0.kml), KML
- [Shapefile](http://data.dentontxgis.opendata.arcgis.com/datasets/fcdc95d116c44587bb835bf3bbdfd56f_0.zip), ZIP

**Comment by abthabib** [comment]
How often is this updated? Is it supposed to be used by charity working on this areas? Can we have a status to this dataset? 'I don't know why it has tags for income, percent.... though they are not part of the dataset.

**Comment by Hoturu** [comment]
I agree better information on what this is showing would be advantageous. It sounds like a listing of repairs done after damage from an emergency service. Some of these are tagged 'optional' so it might include just repairs done by homeowners. There are only 360 entries and the data goes back to 1985.
